js/watermark.js

```diff
@@ -0,0 +1,54 @@
+  
+  wave = "m48.736 55.595l0.419 0.403c11.752 9.844 24.431 8.886 34.092 2.464 6.088-4.049 33.633-22.367 49.202-32.718v-10.344h-116.03v27.309c7.071-1.224 18.47-0.022 32.316 12.886m43.651 45.425l-13.705-13.142c-1.926-1.753-3.571-3.04-3.927-3.313-11.204-7.867-21.646-5.476-26.149-3.802-1.362 0.544-2.665 1.287-3.586 1.869l-28.602 19.13v34.666h116.03v-24.95c-2.55 1.62-18.27 10.12-40.063-10.46m-44.677-42.322c-0.619-0.578-1.304-1.194-1.915-1.698-13.702-10.6-26.646-5.409-29.376-4.116v11.931l6.714-4.523s10.346-7.674 26.446 0.195l-1.869-1.789m16.028 15.409c-0.603-0.534-1.214-1.083-1.823-1.664-12.157-10.285-23.908-7.67-28.781-5.864-1.382 0.554-2.7 1.303-3.629 1.887l-13.086 8.754v12.288l21.888-14.748s10.228-7.589 26.166 0.054l-0.735-0.707m68.722 12.865c-4.563 3.078-9.203 6.203-11.048 7.441-4.128 2.765-13.678 9.614-29.577 2.015l1.869 1.797c0.699 0.63 1.554 1.362 2.481 2.077 11.418 8.53 23.62 7.303 32.769 1.243 1.267-0.838 2.424-1.609 3.507-2.334v-12.234m0-24.61c-10.02 6.738-23.546 15.833-26.085 17.536-4.127 2.765-13.82 9.708-29.379 2.273l1.804 1.729c0.205 0.19 0.409 0.375 0.612 0.571l-0.01 0.01 0.01-0.01c12.079 10.22 25.379 8.657 34.501 2.563 5.146-3.436 12.461-8.38 18.548-12.507l-0.01-12.165m0-24.481c-14.452 9.682-38.162 25.568-41.031 27.493-4.162 2.789-13.974 9.836-29.335 2.5l1.864 1.796c1.111 1.004 2.605 2.259 4.192 3.295 10.632 6.792 21.759 5.591 30.817-0.455 6.512-4.351 22.528-14.998 33.493-22.285v-12.344";
+
+  // put legend in the Atlantic
+  var xPosWatermark = chart_width*0.01,
+      yPosWatermark = 0.95*chart_height;
+  
+  // add group for legend elements & move into the Atlantic Ocean
+var watermark = svg
+      .append('g')
+        .attr('transform', 
+              'translate('+xPosWatermark+','+yPosWatermark+')scale(0.25)')
+        .append('a')
+        .attr("xlink:href", "https://www2.usgs.gov/water")
+        .attr("target", "_blank");
+
+    watermark
+      .append('rect')
+        .attr("x", 0)
+        .attr("y", 0)
+        .attr("width", 500)
+        .attr("height", 140)
```

Comment:
You have `watermarkHeight` and `watermarkWidth` as variables at the top, so I think you'd probably want to use them here? Also, x and y should probably also use the `xPosWatermark` and `yPosWatermark` so that you only need to change the numbers in one place if you want to move it